File: src/GLFMmatlab/mapping_functions/df_p_1.m
function y = df_p_1(x, mu, w)
    % derivative of transformation function for positive data
    % X -> Y (from data to pseudo-obversations)
    if (w == 0)
        error('scaling factor should never be 0');
    end
    y = exp(w.*x) ./ ( exp(w.*(x-mu) - 1) );

File: src/GLFMR/df_p_1.R

df_p_1<-function(x, mu, w){
  if(w == 0){
    stop('scaling factor should never be 0')
  }
  else{
    y <- ( w* exp(w*(x-mu)) ) / as.vector(exp(w*(x-mu) - 1))
  }
  return(y)
  } 
